OpenEdX/Managers/AnalyticsManager/AnalyticsManager.swift

```diff
@@ -55,13 +55,15 @@ class AnalyticsManager: AuthorizationAnalytics,
             analyticsServices.append(segmentService)
         }
         
+        #if PROD
```

Comment:
Could you please add a comment why we are doing this for clarity?

Reply:
We only want to record FullStory events for the `PROD` environment for the following reasons:

1. `Dev` & `Stage` environments has `orgID` of Test Organization named `2U - Mobile Apps`, and we do not want to record events or other data for this test organization.
2. Initially, we set up conditional loading for the FullStory SDK, but it caused issues with enabling SwiftUI-based views in FullStory sessions. We reached out to FullStory's technical support team, who informed us that conditional integration of the FullStory SDK is not possible. As a workaround, we used the test organization `2U - Mobile Apps` and its `orgID` for the `Dev` and `Stage` environments to avoid disrupting the SDK functionality. We have also communicated our usage strategy to FullStory's support team and requested a more effective solution in future SDK updates.